rhc-canonical-facts.timer

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=yggdrasil canonical-facts service timer
+Documentation=https://github.com/RedHatInsights/yggdrasil
+
+[Timer]
+OnCalendar=hourly
+Persistent=true
+RandomizedDelaySec=600
```

Comment:
Why do you randomize this timer? It only write facts to some file. It does not send anything to any server. It could not cause some burst traffic. You only have to wait about 10 minutes for result after starting the timer.

Reply:
That's leftover from the unit files I copied this from. I can drop it.